### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by ItsSynchro** [Ghost-chu/QuickShop-Reremake#1552]
### Description

When a shop is "Out of Stock" the seller will receive a jumbled up message for that item. 

### Steps to reproduce

This happens whilst having a default fresh setup along with a modified messages.yml

### Expected Behaviour

Have it show the actual item that has run out of stock instead of the jumbled up text.

### Screenshots

![unknown](https://user-images.githubusercontent.com/16171807/139957043-3db0b8d9-59bf-4dd6-9f00-3d2bfa1ad96f.png)

### `/qs paste` URL

https://paste.helpch.at/ituqofeqez

**Comment by Ghost-chu** [comment]
Make sure your override file is valid JSON format.

**Comment by ItsSynchro** [comment]
May I know how I'd go about doing that? This occurs during a fresh install of Quickshop 

I've gone ahead and made a video installing it and it occurring...
https://youtu.be/hv3JyANkm8g

**Comment by ItsSynchro** [comment]
> Make sure your override file is valid JSON format.

May I know how I'd go about doing that? This occurs during a fresh install of Quickshop 

I've gone ahead and made a video installing it and it occurring...
https://youtu.be/hv3JyANkm8g

**Comment by Ghost-chu** [comment]
> > Make sure your override file is valid JSON format.
> 
> May I know how I'd go about doing that? This occurs during a fresh install of Quickshop
> 
> I've gone ahead and made a video installing it and it occurring... https://youtu.be/hv3JyANkm8g

Turn on the dev-mode `/qs debug`, and turn off the DisplayAutoDespawn feature (that filled up whole debug logs with useless garbage) , try to reproduce then create the paste again.

**Comment by Ghost-chu** [comment]
> ### Description
> When a shop is "Out of Stock" the seller will receive a jumbled up message for that item.
> 
> ### Steps to reproduce
> This happens whilst having a default fresh setup along with a modified messages.yml
> 
> ### Expected Behaviour
> Have it show the actual item that has run out of stock instead of the jumbled up text.
> 
> ### Screenshots
> ![unknown](https://user-images.githubusercontent.com/16171807/139957043-3db0b8d9-59bf-4dd6-9f00-3d2bfa1ad96f.png)
> 
> ### `/qs paste` URL
> https://paste.helpch.at/ituqofeqez
> 
> ### Additional Context
> _No response_

Also the text with `!-!-!` is send to you while you joined game or playing?

(join message or live message during play)

**Comment by Ghost-chu** [comment]
okay I can reproduce that while run out of stock

**Comment by Ghost-chu** [comment]
for devs:
```
Sending debug: "TextComponent{text\u003d, BaseComponent(color\u003d, font\u003dnull, bold\u003dnull, italic\u003dnull, underlined\u003dnull, strikethrough\u003dnull, obfuscated\u003dnull, insertion\u003dnull, extra\u003d[TextComp3dnull, underlined\u003dnull, strikethrough\u003dnull, obfuscated\u003dnull, insertion\u003dnull, extra\u003d[TextComponent{text\u003dGhost_chu purchased 1 , BaseComponent(color\u003d, font\u003dnull, bold\u003dnull, italic\u003dnull, underlined\u003dnull, strikethrough\u003dnull, obfuscated\u003dnull, insertion\u003dnull, extra\u003dnull, clickEvent\u003dnull, hoverEvent\u003dnull)}, TranslatableComponent(format\u003d%(?:(\\d+)\\$)?([A-Za-z%]|$), translate\u003dblock.minecraft.chest, with\u003dnull), TextComponent{text\u003d from your shop, and you earned 1.0.\nYour shop at -173, 70, 751 has run out of !-!-!\u003d-\u003d-\u003d-\u003d-\u003d-\u003deyJ0cmFuc2xhdGUiOiJibG9jay5taW5lY3JhZnQuY2hlc3QifQ\u003d\u003d\u003d-\u003d-\u003d-\u003d-\u003d-\u003d!-!-!!, BaseComponent(color\u003d, font\u003dnull, bold\u003dnull, italic\u003dnull, underlined\u003dnull, strikethrough\u003dnull, obfuscated\u003dnull, insertion\u003dnull, extra\u003dnull, clickEvent\u003dnull, hoverEvent\u003dnull)}], clickEvent\u003dnull, hoverEvent\u003dHoverEvent(action\u003dSHOW_ITEM, contents\u003d[Text(value\u003d[Lnet.md_5.bungee.api.chat.BaseComponent;@53b0c6b1)], legacy\u003dtrue))}"
```

**Comment by Ghost-chu** [comment]
should fixed in 5.0.0.7
